docs/workflows/api-token.md

```diff
@@ -0,0 +1,400 @@
+data=data,
+auth=('MANAGEMENT_API_CLIENT_ID', 'MANAGEMENT_API_CLIENT_SECRET'),
+)
+```
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go title="/token"
+package main
+
+import (
+"fmt"
+"io/ioutil"
+"log"
+"net/http"
+"strings"
+)
+
+func main() {
+client := &http.Client{}
+var data = strings.NewReader(`grant_type=client_credentials&scope=SCOPES`)
+req, err := http.NewRequest("POST", "https://auth-REGION.beyondidentity.com/v1/tenants/TENANT_ID/realms/REALM_ID/applications/MANAGEMENT_APPLICATION_ID/token", data)
+if err != nil {
+  log.Fatal(err)
+}
+req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+req.SetBasicAuth("MANAGEMENT_API_CLIENT_ID", "MANAGEMENT_API_CLIENT_SECRET")
+resp, err := client.Do(req)
+if err != nil {
+  log.Fatal(err)
+}
+defer resp.Body.Close()
+bodyText, err := ioutil.ReadAll(resp.Body)
+if err != nil {
+  log.Fatal(err)
+}
+fmt.Printf("%s\n", bodyText)
+}
+```
+
+</TabItem>
+</Tabs>
+
+### Authorization Code Flow:
+
+If you would like to use the authorization code flow, use the following curl examples below to optain an authorization code and then create a token with that code.
+
+1. Authenticate to optain an authorization code:
+
+<Tabs groupId="api-token-platform" queryString>
+<TabItem value="curl" label="Curl">
+
+```bash title="/authorize"
+curl https://auth-$REGION.beyondidentity.com/v1/tenants/$TENANT_ID/realms/$REALM_ID/applications/$MANAGEMENT_APPLICATION_ID/authorize?response_type=code&client_id=$MANAGEMENT_API_CLIENT_ID&redirect_uri=$REDIRECT_URI&scope=openid&state=$STATE&code_challenge=$OPTIONAL_CODE_CHALLENGE&code_challenge_method=S256
+```
+
+</TabItem>
+<TabItem value="node" label="Node">
+
+```jsx title="/authorize"
+fetch(
+  'https://auth-REGION.beyondidentity.com/v1/tenants/TENANT_ID/realms/REALM_ID/applications/MANAGEMENT_APPLICATION_ID/authorize?response_type=code'
+);
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python title="/authorize"
+import requests
+
+params = {
+  'response_type': 'code',
+}
+
+response = requests.get(
+  'https://auth-REGION.beyondidentity.com/v1/tenants/TENANT_ID/realms/REALM_ID/applications/MANAGEMENT_APPLICATION_ID/authorize',
+  params=params,
+)
+```
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go title="/authorize"
+package main
+
+import (
+"fmt"
+"io/ioutil"
+"log"
+"net/http"
+)
+
+func main() {
+client := &http.Client{}
+req, err := http.NewRequest("GET", "https://auth-REGION.beyondidentity.com/v1/tenants/TENANT_ID/realms/REALM_ID/applications/MANAGEMENT_APPLICATION_ID/authorize?response_type=code", nil)
+if err != nil {
+  log.Fatal(err)
+}
+resp, err := client.Do(req)
+if err != nil {
+  log.Fatal(err)
+}
+defer resp.Body.Close()
+bodyText, err := ioutil.ReadAll(resp.Body)
+if err != nil {
+  log.Fatal(err)
+}
+fmt.Printf("%s\n", bodyText)
+}
+```
+
+</TabItem>
+</Tabs>
+
+2. Create an access token the with authorization code:
```

Comment:
I think we should somehow clarify that creating an access token happens in the callback - the thing that serves redirect_uri. I'm not sure how exactly we might want to write that, but I see some options:

1. Have a separate section about callback and what needs to be done there (extract code from request, check state etc.)
2. Start this section by saying that this needs to happen in callback where we receive the grant code